scripts/tools/metric.yml

```diff
@@ -115,10 +118,21 @@ etors:
       desc:
             "1.6": "Metric type: instruction pointer"
             "1.7": "Metric type: instruction pointer. Deprecated, use $T_METRIC_TYPE_IP."
-      value: "0x7ffffffe"
     - name: IP
       desc: "Metric type: instruction pointer"
       version: "1.7"
+    - name: EXP_EVENT_NO_VALUE
```

Comment:
I think these need to have values set from 0x7ffffffe, so that compatibility is not broken in the future.

Reply:
you mean EXP_EVENT_NO_VALUE have value 0x7ffffffe +1 and so on? 
I see your point on compatibility, but if the user is truly using enums numbers instead of the entry name it seems to me the user is breaking a fundamental benefit of using enums.